libs/luci-lib-nixio/src/tls-socket.c

```diff
@@ -202,20 +101,17 @@ static int nixio_tls_sock_send(lua_State *L) {
 static int nixio_tls_sock_accept(lua_State *L) {
 	SSL *sock = nixio__checktlssock(L);
 	const int stat = SSL_accept(sock);
-	nixio_tls__set_connected(L, stat == 1);
 	return nixio__tls_sock_pstatus(L, sock, stat);
 }
 
 static int nixio_tls_sock_connect(lua_State *L) {
 	SSL *sock = nixio__checktlssock(L);
 	const int stat = SSL_connect(sock);
-	nixio_tls__set_connected(L, stat == 1);
 	return nixio__tls_sock_pstatus(L, sock, stat);
 }
 
 static int nixio_tls_sock_shutdown(lua_State *L) {
 	SSL *sock = nixio__checktlssock(L);
-	nixio_tls__set_connected(L, 0);
 	return nixio__tls_sock_pstatus(L, sock, SSL_shutdown(sock));
 }
 
```

Comment:
There's another `#ifdef WITH_AXTLS` block in lines 123-125 (orig: 227-229) below that needs to be taken care of.